taskcluster/scripts/build/linux.sh

```diff
@@ -52,24 +56,32 @@ fi
 sudo apt-get update
 
 # Find and extract the package source
-DSCFILE=$(find ${MOZ_FETCHES_DIR} -name '*.dsc' | grep -E -- "-${DIST}[0-9]+.dsc")
+DSCFILE=$(find ${MOZ_FETCHES_DIR} -name '*.dsc')
 if [[ ! -f "$DSCFILE" ]]; then
-    echo "Unable to locate DSC file to build for ${DIST}" >&2
+    echo "Unable to locate DSC file" >&2
     echo "${MOZ_FETCHES_DIR} contained:" >2&
-    find ${MOZ_FETCHES_DIR} -name '*.dsc' >2&
+    ls -al ${MOZ_FETCHES_DIR}
     exit 1
 fi
 dpkg-source -x ${DSCFILE} $(pwd)/mozillavpn-source/
 DPKG_PACKAGE_SRCNAME=$(dpkg-parsechangelog -l mozillavpn-source/debian/changelog -S Source)
-DPKG_PACKAGE_VERSION=$(dpkg-parsechangelog -l mozillavpn-source/debian/changelog -S Version)
-DPKG_PACkAGE_BINNAME=$(cat mozillavpn-source/debian/control | grep "^Package:" | awk '{print $2}')
+DPKG_PACKAGE_BASE_VERSION=$(dpkg-parsechangelog -l mozillavpn-source/debian/changelog -S Version)
+DPKG_PACKAGE_DIST_VERSION=${DPKG_PACKAGE_BASE_VERSION}-${DIST}1
+
+# Update the changelog to release for the target distribution.
+# TODO: If this came from a PR, we should put the author's info here.
```

Comment:
Can you link a ticket for that ? :)

Reply:
Perhaps it's faster for me to just implement it.